cafebabel/templates/articles/_card.html

```diff
@@ -1,5 +1,5 @@
 <article>
-  {% if article.has_image %}
+  {% if article.image_filename %}
```

Comment:
Because we don't use the `image_filename` property directly, I suppose it could be more explicit and generic to either reimplement a pseudo property `has_image` or write a method `has_image()`.